Rollbar.NetPlatformExtensions/RollbarLogger.cs

```diff
@@ -203,52 +202,44 @@ mslogging.EventId eventId
 
         private bool disposedValue = false; // To detect redundant calls
 
+
         /// <summary>
         /// Releases unmanaged and - optionally - managed resources.
         /// </summary>
         /// <param name="disposing">
         ///   <c>true</c> to release both managed and unmanaged resources; 
         ///   <c>false</c> to release only unmanaged resources.
         /// </param>
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Major Code Smell", "S1066:Collapsible \"if\" statements should be merged", Justification = "Cleaner code structure.")]
         protected virtual void Dispose(bool disposing)
         {
             if (!disposedValue)
             {
                 if (disposing)
                 {
-                    // TODO: dispose managed state (managed objects).
+                    // dispose managed state (managed objects).
 
                     if (this._rollbar != RollbarLocator.RollbarInstance)
                     {
                         this._rollbar.Dispose();
                     }
                 }
 
-                // TODO: free unmanaged resources (unmanaged objects) and override a finalizer below.
-                // TODO: set large fields to null.
+                // free unmanaged resources (unmanaged objects) and override a finalizer below.
+                // set large fields to null.
```

Comment:
Same here as above. Is this supposed to be a TODO, or is the thing in the comment already implemented now?

Reply:
this was already addressed by my comment/reply above...